### ISSUE TITLE
Accessibility: Missing 'alt text' for images

### ISSUE THREAD

**Issue by Dawnzar** [CityofOttawa/ckanext-ottawa#4]
WCAG 2.0 Success Criteria 1.1.1 - Non text Content Level (A) Compliance violation - All images must have equivalent alternative text.
1. Go to: data.ottawa.ca
2. Observe for alt text on the images within the 'Outdoor Pools' and 'Parks and Volleyball Courts' images.

Resolution: Add alternative text to images.
missing alt text images
![missing alt text images](https://cloud.githubusercontent.com/assets/5271253/8012432/774803b6-0b8f-11e5-98da-8b8420c3b039.JPG)
![volleyball courts alt text](https://cloud.githubusercontent.com/assets/5271253/8012437/817ec482-0b8f-11e5-99f3-b536beba2c51.JPG)
